.buildkite/scripts/release/redhat-preflight.sh

```diff
@@ -20,7 +22,8 @@ container_already_verified() {
 }
 
 main() {
-    local tag="${1#v}"
+    local tag
+    tag=$(cat "$ROOT"/VERSION)
```

Comment:
It's not you, but reading this code again, I don't like how I wrote it.
Your change will work, but the code suggests that `tag` is local but it still used by the other function `container_already_verified`.

Maybe remove the local work and use uppercase to suggest it's global? Rename it to TAG or ECK_VERSION?
```bash
    ECK_VERSION=$(cat "$ROOT/VERSION")
    export ECK_VERSION
```

Or don't use export but declare the vars outside main:

```
API_KEY=
PROJECT_ID=
ECK_VERSION=
```